src/main/resources/assets/ragi_materials/lang/ja_jp.lang

```diff
@@ -24,7 +24,6 @@ item.unfired_cast_2.name=未焼成の粘土型 (歯車)
 item.unfired_cast_3.name=未焼成の粘土型 (板材)
 item.unfired_cast_4.name=未焼成の粘土型 (棒材)
```

Comment:
Suggestion:
```suggestion
item.ragi_materials.book_respawn.name=[開発用] リスポーン本
item.ragi_materials.cast_gear.name=粘土型 (歯車)
item.ragi_materials.cast_ingot.name=粘土型 (インゴット)
item.ragi_materials.cast_nugget.name=粘土型 (ナゲット)
item.ragi_materials.cast_plate.name=粘土型 (板材)
item.ragi_materials.cast_stick.name=粘土型 (棒材)
item.ragi_materials.crushing_hammer.name=破砕ハンマー
item.ragi_materials.unfired_cast_0.name=未焼成の粘土型 (インゴット)
item.ragi_materials.unfired_cast_1.name=未焼成の粘土型 (ナゲット)
item.ragi_materials.unfired_cast_2.name=未焼成の粘土型 (歯車)
item.ragi_materials.unfired_cast_3.name=未焼成の粘土型 (板材)
item.ragi_materials.unfired_cast_4.name=未焼成の粘土型 (棒材)
```
to avoid possible key crush issues.